.gitignore

```diff
@@ -34,3 +34,4 @@ intellij-plugin/djinni.jar
 
 # profiling output
 callgrind.out.*
+src/
```

Comment:
Can you explain this?  Ignoring the src/ directory doesn't seem obviously right.

Reply:
You are completely right. I guess this is a wrong manipulation on my side during a find/replace sequence.